aten/src/ATen/native/cudnn/Conv.cpp

```diff
@@ -1106,11 +1106,9 @@ Tensor cudnn_convolution_transpose(
     IntArrayRef padding, IntArrayRef output_padding, IntArrayRef stride, IntArrayRef dilation,
     int64_t groups, bool benchmark, bool deterministic)
 {
-
   TensorArg input  { input_t,  "input",  1 },
             weight { weight_t, "weight", 2 };
-  CheckedFrom c = "cudnn_convolution_transpose";
-  
+  CheckedFrom c = "cudnn_convolution_transpose"; 
```

Comment:
This patch is relatively simple, so it's not a big deal for the reviewer, but as a general note, try to avoid extraneous changes like this in your diffs. It interferes with git blame and adds line noise the reviewer has to wade through when doing the review.